app/helpers/user_notifications_helper.rb

```diff
@@ -21,8 +21,9 @@ def correct_top_margin(html, desired)
   end
 
   def logo_url
-    logo_url = SiteSetting.site_digest_logo_url
-    logo_url = SiteSetting.site_logo_url if logo_url.blank? || logo_url =~ /\.svg$/i
+    dark_enabled = SiteSetting.dark_mode_emails_active
+    logo_url = SiteSetting.public_send("site_digest_logo#{'_dark' if dark_enabled}_url")
+    logo_url = SiteSetting.public_send("site_logo#{'_dark' if dark_enabled}_url") if logo_url.blank? || logo_url =~ /\.svg$/i
```

Comment:
I don't think you can use `dark_enabled` here because we don't know whether the user's email client is in dark mode.   In the current state, the digest will always output the dark logo when the site has `dark_mode_emails_active` enabled, meaning, even clients in regular light mode will get the dark logo. 

In core, when auto dark mode is enabled, the HTML output includes markup for two logos: a light one and a dark one, and we use CSS media queries to pick the right one. I don't know whether email CSS supports that...

For now, I think it's fine to have just the regular digest logo in the first iteration of this feature.